### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by zijiancogito**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -10,7 +10,7 @@ might be the analysis of the ``file`` program from
 
 # Prerequisites
 
-We assume that you are working on an Ubuntu 18.04 LTS 64-bit OS.
+We assume that you are working on an Ubuntu 20.04 LTS 64-bit OS.
```
Comment (vitalych) [inline README.md:13]: I would rephrase it like this: `We assume that you are working on the latest LTS release of Ubuntu on a 64-bit OS.`.
Otherwise, we'll need to keep updating this forever.

**Review by vitalych — CHANGES_REQUESTED** [review]
Also change the commit message to: `readme: updated OS requirements`

**Comment by 0xk4n3ki** [comment]
lol, now its 22.04 :)
